### ISSUE TITLE
procs display issue

### ISSUE THREAD

**Issue by zlscherr** [dalance/procs#92]
When running procs on my Mac Catalina, the display seems to be separated by many lines

<img width="1680" alt="Screen Shot 2020-11-24 at 3 03 56 PM" src="https://user-images.githubusercontent.com/15149445/100145689-b0a3d180-2e66-11eb-9291-264900b7dbd4.png">

Any idea what's going on? If I do

procs | cat

then the spacing appears to be normal.  Also, if I just run procs --watch then the spacing is also fine

<img width="1678" alt="Screen Shot 2020-11-24 at 3 08 53 PM" src="https://user-images.githubusercontent.com/15149445/100145882-00829880-2e67-11eb-8439-39047ec35474.png">

Thanks,
Zach

**Comment by dalance** [comment]
Thank you for your report.
It seems to be pager problem.

Could you try the following commands?

```console
$ procs --pager disable
$ procs --color always | less -SR
```

**Comment by zlscherr** [comment]
Both of those commands display successfully.  As a temporary workaround I will just alias procs to one of those options.

I'm not sure what's happening with the pager, I unset PAGER and LESS but with just running procs the problem persists.  Thank you again for your help.

**Comment by dalance** [comment]
Maybe,  can `procs --color always | more -f` reproduce the issue?
If so, `less` detection is failed.

Could you try to put the following file?

```
$ cat ~/.config/procs/config.toml
[pager]
command = "less -SR"
```

**Comment by roblav96** [comment]
<img width="1080" alt="Screen Shot 2020-11-24 at 9 31 45 PM" src="https://user-images.githubusercontent.com/1457327/100175489-7a814480-2e9c-11eb-8471-fe0cd90df58f.png">

Same issue here.

**Comment by zlscherr** [comment]
I did some git bisecting and it seems this problem started in 072e7b026e79a773641fc2c94143f470fb9d527c.  The problem seems to be in src/view.rs in the lines:

`            Pager::with_pager("less -SR")
                .pager_envs(&["LESSCHARSET=utf-8"])
                .setup();
`

I tried removing the .pager_envs(&["LESSCHARSET=utf-8"]) line and it worked correctly.  I don't really know anything about pagers, but somehow this line is leading to problems.

**Comment by zlscherr** [comment]
And also it's not specifically the LESSCHARSET=utf-8 that causes problems.  If I change the line to something like

pager_envs(&["X=3"])

then the display is still weird.  Maybe it's a problem with the pager crate?

**Comment by dalance** [comment]
Thank you for a lot of information.
I'll disable it on macOS because it is a workaround for Ubuntu.

**Comment by zlscherr** [comment]
You're welcome.  I also fooled around a bit, and if I change the line to

`Pager::with_pager("/usr/bin/less -SR")
                .pager_envs(&["LESSCHARSET=utf-8"])
                .setup();`

then it seems to work fine.  I don't know enough rust to know what's going on, but I'm happy to test anything on Mac.

**Comment by roblav96** [comment]
@zlscherr
In that screenshot, my `procs.toml` has `pager` set to `never` and zsh env var `PAGER=cat`

**Comment by dalance** [comment]
Could you try https://github.com/dalance/procs/commit/b90b4ccf15e5806aff6f5306e2b9f81e2921de34?
If it works fine, I'll release it as v0.10.10.

**Comment by zlscherr** [comment]
That fixes the issue for me.  Thank you so much, and thank you for designing such a useful tool!

**Comment by dalance** [comment]
Thanks. I released v0.10.10.
